packages/app/src/systems/Swap/machines/swapMachine.typegen.ts

```diff
@@ -0,0 +1,162 @@
+// This file was automatically generated. Edits will be overwritten
```

Comment:
If this is `build-time` generation we should remove from track

Reply:
✅